proto/nibiru/oracle/v1/event.proto

```diff
@@ -8,11 +8,45 @@ import "google/api/annotations.proto";
 option go_package = "github.com/NibiruChain/nibiru/x/oracle/types";
 
 // Emitted when a price is posted
-message OraclePriceUpdate {
+message EventExchangeRateUpdate {
   string pair = 1;
-  string price = 2 [
+  string exchange_rate = 2 [
```

Comment:
(my two cents) TBH I'm not a fan of the `Exchange Rate` terminology. It's a remnant of Terra's oracle where they had exchange rates between USD/KRW/EUR/JPY paired against LUNA. Not a big deal though.

Reply:
Yes, I prefer price as well. Update in progress...

Reply:
Fixed by https://github.com/NibiruChain/nibiru/commit/8dc9837fa945b5f62cd6b7e8658bee709910a35e